src/api/theMovieDB.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Fetch movie's data from themoviedb api
+ *
+ * @param {number} page - Number of the page to query
+ * @returns {object}
+ */
+async function getMovies(page) {
+  const respnse = await fetch(
+    `https://api.themoviedb.org/3/discover/movie?api_key=${process.env.MOVIE_API_KEY}&page=${page}`
+  );
+
+  const result = await respnse.json();
```

Comment:
you can just return without creating the variable

Reply:
Yep agree on this 😅